server.js

```diff
@@ -17,11 +16,6 @@ async function init(environment) {
   server.app.services = services;
 
   // Extend Hapi response toolkit and request interfaces.
-  // eslint-disable-next-line prefer-arrow-callback
-  server.decorate('toolkit', 'error', function error(name, ...args) {
-    throw Boom[name](...args);
-  });
-
```

Comment:
You could have used it in your error handling code:

```javascript
function handleError(h, { message }) {
  switch (message) {
    case 'invalid-url':
      return h.error('badData', message);
    // …
  }
}
```

This would allow you to use all of the Hapi toolkit functionality in your error handler and if you decide to do more with these errors, to just change it in one place. Might not be needed at the moment though, so 🤷‍♂️